### PR TITLE
[Feature] Owners

### PR DESCRIPTION
**Pull request by hzoo**
Lets finish this up! fixed merge conflicts

### CODE REVIEW

File: src/commands/OwnersCommand.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
+import NpmUtilities from "../NpmUtilities";
+import Command from "../Command";
+import async from "async";
+
+export default class LsCommand extends Command {
```
Comment (doug-wade) [inline src/commands/OwnersCommand.js]: OwnersCommand?

**Comment by thejameskyle** [comment]
This is missing pretty much the whole implementation

**Comment by hzoo** [comment]
Ah, yeah I just rebased with your wip branch. we'll want to use https://github.com/cloudflare/cf-ui/tree/master/scripts right?

File: src/commands/InitCommand.js [hunk 2]
```diff
@@ -74,7 +80,8 @@ export default class InitCommand extends Command {
 
     objectAssign(lernaJson, {
       lerna: this.lernaVersion,
-      version: version
+      version: version,
+      owners: owners
```
Comment (rygine) [inline src/commands/InitCommand.js]: could this just be `owners: this.repository.owners`?

or destructured in above, just `owners`.

**Pull request closed without merging**